content/coordination/meetup/index.en.md

```diff
@@ -11,6 +11,25 @@ platform. The person in charge needs to be able to commit a few hours weekly.
 Ideally, this will become a team of at least 2 people, to split the workload
 and guarantee requests are addressed any time of the year.
 
+## Why do we use Meetup?
+
+Meetup might not be well-known to you or your region, but it has been a very 
```

Comment:
Please add line breaks after each sentence. :pray:

Reply:
on my end it looks like there are line breaks??